cves/CVE-2017-14056.yml

```diff
@@ -342,7 +360,16 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The main mistake is trusting inputs. The chance of an attacker
+    getting someone to both download and then use an evil file just for 
+    a Denial of Service attack are low, so it is easy to see why this may have
+    been overlooked when the feature was first added. This issue is seen in other
+    demuxers as well, which demonstrates either a lack of testing and understanding
+    around file headers, or a lack of requirements around processing headers. 
+    RL2 files are a very niche video format, as they come from the adult video game
+    "Voyeur", so it is very likely the developer was not aware the header frame count
```

Comment:
I like how you explain the file is niche, which can be a large reason as to why this wasn't found. I think you could also take this "adult" context and make it into a funny (and appropriate) nickname.

Nice job overall! 😄

Reply:
+3 upvotes

Interesting/niche codec, and while EOF checks are basic this is quite easy to understand yet interesting, as CPU slowdowns are certainly important.

Reply:
Thanks for the review, super helpful. I fixed up most of the grammar and nitpicks, and added a decent amount to the description, including what a demuxer is.